internals/webpack/webpack.base.babel.js

```diff
@@ -163,8 +163,7 @@ module.exports = options => ({
       '@themes': path.resolve(__dirname, '../../app/themes'),
       '@images': path.resolve(__dirname, '../../app/images'),
       '@hooks': path.resolve(__dirname, '../../app/hooks'),
-      moment$: path.resolve(__dirname, '../../node_modules/moment/moment.js'),
-      '@ant-design/icons/lib/dist$': path.resolve(__dirname, './app/icons.js')
```

Comment:
what was the purpose of removing this?

Reply:
I didn't removed it, maybe its removed while building